test/core_registration_test.py

```diff
@@ -655,7 +655,29 @@ def test_active_inactive_remove_module(self):
         self.assertEqual(str(e.exception),
             'Cannot delete module that has been released.  Make it inactive instead.');
 
+    def test_extra_files(self):
+        # (1) register the test repo
+        giturl = self.cUtil.get_test_repo_1()
```

Comment:
Where does cUtil come from?

Reply:
It's defined in the setup class L669

Reply:
Sorry, I'm having trouble figuring this out. What setup class?

Reply:
https://github.com/JamesJeffryes/catalog/blob/f435e20abf6450c08a43b629d7bee2258c52861b/test/core_registration_test.py#L691